sdk/tables/ci.yml

```diff
@@ -30,7 +30,7 @@ extends:
       CtestRegex: azure-data-tables.*
       LineCoverageTarget: 77
       BranchCoverageTarget: 42
-      LiveTestCtestRegex: azure-data-tables
+      LiveTestCtestRegex: azure-data-tables.*
```

Comment:
Even if it works without `.*`, I like that lines 30 and 33 are the same now.

Reply:
100% agreed with Anton.